src/inverseRobot.js

```diff
@@ -24,7 +24,24 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const keys = Object.keys(robot);
+  const values = Object.values(robot);
+  const res = {};
```

Comment:
`res` isn't informative 
```suggestion
  const newRobot = {};
```
```suggestion
  const inversedRobot = {};
```